source/edge-server/manage-edge-servers.txt

```diff
@@ -0,0 +1,518 @@
+      instance, including the date and time of the last connection, and a 
+      snapshot of its status. You can optionally view details or logs for each 
+      instance.
+
+      To view the Edge Server dashboard, choose :guilabel:`Edge Server` in
+      the left navigation sidebar.
+
+      To view the details for a specific Edge Server instance, click 
+      :guilabel:`View Details` for that Edge Server instance. Alternately,
+      click the :guilabel:`App Logs` button to browse logs associated with 
+      the instance.
+
+   .. tab::
+      :tabid: api
+
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/api-authenticate-instructions.rst
+
+         .. step:: Get the List of Edge Server Instances
+
+            Call the <LIST EDGE SERVER> endpoint to get a list of all the 
+            Edge Servers instances:
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+            Find the ID of the instance whose details you want to view.
+
+         .. step:: Call the Edge Server Info Endpoint
+
+            Call the <GET EDGE SERVER INFO> endpoint with the ID of the 
+            instance whose details you want to view.
+
+            .. code-block:: shell
+
+               INSERT API DETAILS HERE
+
+.. _edge-server-details:
+
+Edge Server Instance Details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The information available in the Edge Server details varies depending on 
+whether the Edge Server has been configured and connected to Atlas, or whether
+configuration and connection are still pending.
+
+.. _edge-server-pending:
+
+Pending Edge Server Instances
+`````````````````````````````
+
+When an Edge Server instance has not yet connected to Atlas for the first time, 
+the instance's :guilabel:`Health` status is :guilabel:`Pending`.
+
+While an Edge Server instance is in :guilabel:`Pending` state, the detail
+view includes:
+
+- A button you can use to rename the instance.
+- An alert noting that you have not completed setting up the instance
+- The instance's ID
+- The Install command that you can use in the terminal to install the 
+  Edge Server code
+- A button to regenerate the registration token
+- A button to delete the Edge Server instance
+
+Once you have successfully configured and connected an Edge Server instance,
+the detail view changes to provide additional details about the instance.
+
+.. _edge-server-active-or-failed:
+
+Active or Failed Edge Server Instances
+``````````````````````````````````````
+
+After you have connected an Edge Server instance to Atlas for the first time, 
+the instance's :guilabel:`Health` status changes to :guilabel:`Active`. 
+
+If the Edge Server instance encounters a fatal error, its :guilabel:`Health`
+status changes to :guilabel:`Failed`.
+
+The details available for active or failed instances include:
+
+- A button you can use to rename the instance.
```

Comment:
same

Reply:
I've updated this to reframe as "actions you can do". If you've got other ideas about how to present this information, I'm happy to revise.